projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/KubernetesService.java

```diff
@@ -2500,4 +2504,8 @@ static int convertMemoryToMBs(Quantity quantity) {
     }
     return quantity.getNumber().toBigInteger().divide(divider.multiply(divider)).intValue();
   }
+
+  public ApiClient getClient() {
```

Comment:
It would only be used by a testing class. 
I think it is better to break encapsulation slightly than add a method used only in testing to this class. 
I've added an annotation to make this more explicit. 
Let me know what you think?